cilium/lb/main.go

```diff
@@ -123,7 +136,7 @@ func init() {
 						Usage: "Delete all entries",
 					},
 				},
-				ArgsUsage: "--all | <address> <port> <slave-index>",
+				ArgsUsage: "--all | <address>:<port>",
```

Comment:
What do you mean? `lb delete-service [IPv6]:port` works fine. Do you mean something different?